src/sig_stfl/lms/external/hss_sign_inc.h

```diff
@@ -22,7 +23,8 @@
  *    hss_sign_update( &ctx, message_part_2, len_2 );
  *    hss_sign_update( &ctx, message_part_3, len_3 );
  *    success = hss_sign_finalize( &ctx, working_key, signature );
- *    if (success) printf( "We generated the signature\n" );
+ *    if (success) printf( "We generated the signature
```

Comment:
Did something go wrong with the `\n` symbol here?

Reply:
Yep, will fix.